src/main/java/com/dynatrace/oneagent/sdk/impl/proxy/OutgoingMessageTracerProxy.java

```diff
@@ -17,7 +17,7 @@
 
 import com.dynatrace.oneagent.sdk.api.OutgoingMessageTracer;
 
-public class OutgoingMessageTracerProxy extends TraceableProxy implements OutgoingMessageTracer {
+final class OutgoingMessageTracerProxy extends TraceableProxy implements OutgoingMessageTracer {
```

Comment:
I recall we talked about something similar before, but why is `public` removed from this class? And why just this class?

Reply:
It's not just this class. It's all *but* `OutgoingWebRequestTracerProxy` which is probably an oversight.
These classes should never have been public (and non-final) in the first place. This will be mentioned in the GH release notes though.

Reply:
There is also `SDK2AgentInternalApiProxy` which is still public.

Reply:
This is actually intentional (or rather: necessary).